pkgs/applications/networking/remote/remmina/default.nix

```diff
@@ -23,7 +24,15 @@ stdenv.mkDerivation (finalAttrs: {
     sha256 = "sha256-oEgpav4oQ9Sld9PY4TsutS5xEnhQgOHnpQhDesRFTeQ=";
   };
 
-  nativeBuildInputs = [ cmake ninja pkg-config wrapGAppsHook ];
+  patches = [
+    (fetchpatch {
+      url = "https://gitlab.com/Remmina/Remmina/-/merge_requests/2525.patch";
```

Comment:
Would you please pin this patch to the commit rather than the merge request to ensure immutability

Reply:
Updated :+1: